src/Bundler/Runner/UnixSocketRunner.php

```diff
@@ -31,24 +32,28 @@ class UnixSocketRunner implements RunnerInterface
     private $config;
     private $socket_location;
     private $factory;
+    private $logger;
     private $start_timeout;
     private $small_timeout;
 
     public function __construct(
         ConfigInterface $config,
         UnixSocketFactory $factory,
-        int $start_timeout = 500000,
+        LoggerInterface $logger = null,
+        int $start_timeout = 5000000,
         int $small_timeout = 100000
     ) {
         $this->config          = $config;
         $this->socket_location = $config->getCacheDir() . '/asset-lib.socket';
         $this->factory         = $factory;
+        $this->logger          = $logger ?? new NullLogger();
```

Comment:
doesn't the logger also exists in the config? `$config->getLogger()`

Reply:
Good point :+1: